packages/app-i18n/src/hooks/useI18N.ts

```diff
@@ -28,18 +28,29 @@ export function useI18N() {
         getLocales() {
             return state.locales;
         },
-        getValue(valueObject?: I18NValueObject): string {
+        getValue(valueObject?: I18NValueObject, locale?: string): string {
             if (!valueObject) {
                 return "";
             }
-
+            locale = locale || self.getLocale().id;
             if (Array.isArray(valueObject.values)) {
-                const output = valueObject.values.find(item => item.locale === self.getLocale().id);
+                const output = valueObject.values.find(item => item.locale === locale);
                 return output ? output.value : "";
             }
 
             return valueObject.value || "";
         },
+        getValues(valueObject?: I18NValueObject): string[] {
```

Comment:
Let's move it into your scope for now. We'll see if this feature is really necessary. If so, we'll add it.